rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/EvaluatorUtil.java

```diff
@@ -194,7 +194,7 @@ private static void extractReasonAndStackTrace(Throwable e, String task, StringW
 
     private enum ErrorHandlingOption {
         // The order in which these values are declared is the order in which they will appear in the VS Code tooltip
-        REPORT_ON_GITHUB("Report on GitHub"),
+        REPORT_ON_GITHUB("Report on GitHub with usethesource/rascal-language-servers"),
```

Comment:
this does make it a very wide button, I don't think we need to be this specific?

Reply:
Well I never recognized this as something that belonged to the Rascal project, because there is no hint to which extension this belongs. I thought it was some co-pilot-like extension of VScode. So I thought let's help the users see that.